wiki/Beatmap/Beatmap_tags/en.md

```diff
@@ -0,0 +1,216 @@
+
+The tables below list all user tags grouped by category as well as the game modes they are applicable in.
+
+### Aim
+
+| Tag name | Description | Game mode |
+| :-- | :-- | :-- |
+| aim/aim control | Patterns with velocity or direction changes which strongly go against a player's natural movement pattern. | ![][osu!] |
+| aim/flow aim | Patterns with fully continuous cursor movement, usually due to a combination of wide angles and little time between objects. | ![][osu!] |
+| aim/jump aim | Focuses heavily on jumps, i.e. circles spaced far apart that require the player to move towards, slow down to hit, then speed up to move towards the next object. | ![][osu!] |
+| aim/linear aim | Patterns, usually jumps, that require the player to move continuously in a straight or nearly straight line. | ![][osu!] |
+| aim/precision | Colloquial term for maps which require fine, precise movement to aim correctly. Typically refers to maps with circle sizes above and including 6. | ![][osu!] |
+| aim/precision | Colloquial term for maps which require fine, precise movement to catch fruits. Typically refers to maps with circle sizes above and including 6. | ![][osu!catch] |
+| aim/sharp aim | Patterns with heavy use of sharp angle movement, either between circles or into sliders. | ![][osu!] |
+| aim/spaced jumps | Focuses heavily on jumps with a lot of spacing. | ![][osu!] |
+| aim/squares | Focuses heavily on jumps that follow square patterns. | ![][osu!] |
+| aim/stamina | Tests a player's ability to aim spaced jumps over long periods of time. | ![][osu!] |
+| aim/triangles | Focuses heavily on jumps that follow triangular patterns. | ![][osu!] |
+| aim/wide aim | Patterns, usually jumps, with heavy use of wide angle movement. | ![][osu!] |
+
+### Geometric
+
+| Tag name | Description | Game mode |
+| :-- | :-- | :-- |
+| geometric/grid snap | Objects are placed along a square grid, typically using osu!'s in-built grid snap feature. | ![][osu!] |
+| geometric/hexgrid | Objects are placed along a hexagonal grid. | ![][osu!] |
+
+### Gimmick
+
+| Tag name | Description | Game mode |
+| :-- | :-- | :-- |
+| gimmick/2B | Includes gameplay elements with two or more objects placed simultaneously. The term originates from a Chinese transliteration of 'idiot'. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| gimmick/barlines | A map that makes use of barlines to enhance visuals or replace notes. | ![][osu!taiko] |
+| gimmick/circle only | Restricts object choice to circles only. | ![][osu!] |
+| gimmick/delay | Maps that feature high snap streams based on the song's delayed sound effect. | ![][osu!mania] |
+| gimmick/dodge the beat | A map where the player needs to avoid every object. | ![][osu!catch] |
+| gimmick/LN inverse | Maps that feature long note holds that emphasize constant holds and releases in quick succession. Most distinct feature of Inverse is the "negative spaces" in the patterning. | ![][osu!mania] |
```

Comment:
"emphasise"?

it seems these tag descriptions haven't quite been checked for british spelling. can handle that later

Reply:
Related to https://github.com/ppy/osu-wiki/pull/13171#issuecomment-2825326809